bulldozer/config_v1.go

```diff
@@ -44,8 +44,9 @@ type MergeConfig struct {
 	DeleteAfterMerge       bool `yaml:"delete_after_merge"`
 	AllowMergeWithNoChecks bool `yaml:"allow_merge_with_no_checks"`
 
-	Method  MergeMethod  `yaml:"method"`
-	Options MergeOptions `yaml:"options"`
+	Method  MergeMethod        `yaml:"method"`
+	Methods []ConditionalMerge `yaml:"merge_method"`
```

Comment:
Calling this `merge_method` in the configuration but `Methods` in the code seems a bit confusing.

Reply:
Changed to `MergeMethods`. I used `merge_method` because of `branch_method`, although `merge_methods` makes more sense. 

Since `branch_method` is going away perhaps `merge_methods`  would be best.